src/region/RegionCreator.tsx

```diff
@@ -23,6 +23,7 @@ export default function RegionCreator({ className, onAbort, onStart, onStop }: P
     const [isDrawing, setIsDrawing] = React.useState<boolean>(false);
```

Comment:
The introduction of two booleans to track this state makes things a little confusing. Can `isDrawing` be true while `hasStarted` is false? Should we be using a `status` enum, instead?